h/static/styles/annotation-share-dialog.scss

```diff
@@ -0,0 +1,130 @@
+.share-dialog-wrapper.open .annotation-share-dialog {
+  display: flex;
+  flex-direction: column;
+}
+
+.annotation-share-dialog {
+  display: none;
+  position: absolute;
+  right: 0;
+  bottom: 130%;
+  z-index: 1;
+  background: $white;
+  border: 1px solid $gray-lighter;
+  border-radius: 2px;
+  width: 200px;
+  font-size: $body1-font-size;
+  cursor: default;
+  box-shadow: 0px 0px 4px 0px rgba(0,0,0,0.15);
+
+  &:after, &:before {
+    top: 100%;
+    right: 10px;
+    border: solid transparent;
+    content: " ";
+    height: 0;
+    width: 0;
+    position: absolute;
+    pointer-events: none;
+  }
+
+  &:after {
+    border-color: rgba(255, 255, 255, 0);
+    border-top-color: $white;
+    border-width: 5px;
+    margin-right: -5px;
+  }
+
+  &:before {
+    border-color: rgba(211, 211, 211, 0);
+    border-top-color: $gray-lighter;
+    border-width: 6px;
+    margin-right: -6px;
+  }
+}
+
+.annotation-share-dialog-msg {
+  color: $gray-light;
+  margin: -5px 10px 10px 10px;
+  line-height: 15px;
+  font-size: 11px;
+}
+
+.annotation-share-dialog-msg__audience {
+  font-style: italic;
+}
+
+.annotation-share-dialog-link {
+  font-size: $body1-font-size;
+  margin: 10px;
+  padding: 5px;
+  color: $text-color;
+  border: 1px solid $gray-lighter;
+  border-radius: 2px;
+  background: $lg-3;
+  white-space: nowrap;
+  overflow: hidden;
+  display: flex;
+  flex-direction: row;
+}
+
+.annotation-share-dialog-link:hover {
+  color: $gray-dark;
+}
+
+.annotation-share-dialog-link__text {
+  overflow: hidden;
+  border: none;
+  width: 100%;
+  background: inherit;
+  height: 22px;
+}
+
+.annotation-share-dialog-link__btn {
+  color: $dg-3;
+  padding: 0px;
+  margin-left: 7px;
+}
+
+.annotation-share-dialog-link__btn:hover {
+  color: $dg-1;
+}
+
+.annotation-share-dialog-target {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  border-bottom: 1px solid $gray-lighter;
+  font-size: $body1-font-size;
+  padding: 10px;
+}
+
+.annotation-share-dialog-target__label {
+  font-weight: bold;
+}
+
+.annotation-share-dialog-target__icon {
+  color: $dg-3;
+  text-decoration: none;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+.annotation-share-dialog-target__icon:hover {
+  color: $dg-1;
+}
+
+.annotation-share-dialog-link__feedback {
+  font-size: 11px;
```

Comment:
What was the status on this? I think the 11px font size was from the palette wasn't it?